slides/04-getting-started-with-k6-oss/slides.md

```diff
@@ -0,0 +1,101 @@
+# Get started with k6 OSS
+
+---
+
+## Installation
+
+https://k6.io/docs/getting-started/installation/ 
+
+![k6 installation page](../../images/installation-page.png)
```

Comment:
This page is strange because I cannot scroll to see the full image.

Reply:
I'll change this image because it is actually cropped 😅